CustomLookup/samplefile.txt

```diff
@@ -0,0 +1,10 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam ut dictum ligula. Aliquam eget viverra sapien. Sed vitae mi sit amet ex laoreet pellentesque eget nec risus. Morbi metus sem, pharetra eu pretium in, mollis et ante. Etiam non leo quis massa finibus vehicula et vitae est. Ut id elementum nisi, nec mollis diam. Suspendisse iaculis, velit eu porta placerat, sem arcu varius est, sit amet tincidunt lectus dolor in nunc. 
```

Comment:
There's a top-level directory for sample files.

Reply:
is this used anywhere?

---
In reply to: [303668365](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r303668365) [](ancestors = 303668365)

Reply:
No sorry,  it was used for testing initially.